activemodel/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+*   Ability to change the default `expires_in` for `has_secure_password` token generator.
+
+    ```ruby
+    class User < ApplicationRecord
+      has_secure_password reset_token_expires_in: 30.minutes
```

Comment:
@abeidahmed the reset token expiry matters only when `reset_token: true` is defined. if it's false, then there's no need for an expires in. imo a better api will be to replace the boolean with an object/hash, similar to how `foreign_key` definition works in migrations. in simple cases, `foreign_key: true` is good enough. when it's difficult for activerecord to figure out the association table the definition becomes `foreign_key: { to_table: :table_name }` instead. maybe we can pursue the same thing here?

```ruby
has_secure_password reset_token: { expires_in: 30.minutes }`
```

Reply:
IMO, this suggestion makes the API better

Reply:
I've implemented your suggestion, @yawboakye.